R/netzschleuder.R

```diff
@@ -210,8 +218,12 @@ ns_df <- function(name, token = NULL, size_limit = 1) {
     nodes_df[["y"]] <- mat[2, ]
   }
 
-  gprops_df <- readLines(unz(temp, gprops_file_name))
+  con_gprops <- unz(temp, gprops_file_name)
+  gprops_df <- readLines(con_gprops)
 
+  on.exit(close(con_edge))
```

Comment:
would it make sense to take a dependency on withr and use `withr::local_connection()`? Or to define a helper since we'd call it at least four times?